src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -407,6 +403,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt.targets" />
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt2.targets" />
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.SkipCases.projitems" />
+<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Tooling.targets" />
```

Comment:
same here 

```
Condition=" '$(UsingAndroidNETSdk)' == 'True' "
```

Reply:
This file is needed everywhere, should I take my own advice?

https://github.com/xamarin/xamarin-android/pull/4459#discussion_r400411074